Gems/Atom/RPI/Code/Source/RPI.Reflect/Material/MaterialAsset.cpp

```diff
@@ -183,6 +194,18 @@ namespace AZ
             m_isDirty = false;
         } 
 
+        void MaterialAsset::ApplyVersionUpdates()
```

Comment:
Required: This kind of process needs to be available for MaterialSourceData as well, so that tools can access the properties by their correct names.